put_get_test.go

```diff
@@ -341,11 +346,23 @@ func TestPutOverwrite(t *testing.T) {
 			t.Fatalf("expected SKIPPED, got %v", s6)
 		}
 
+		rows = dbt.mustQuery("ls @~/test_put_overwrite")
+		defer rows.Close()
+		if rows.Next() {
```

Comment:
Same here. If `rows.Next` returns false, we never update `s3`, so we compare the same value and test passes even it it should not.